.devcontainer/devcontainer.json

```diff
@@ -33,17 +34,15 @@
 		"pmbenjamin.vscode-snyk",
 		"redhat.vscode-yaml",
 		"richie5um2.vscode-sort-json",
-		"tabnine.tabnine-vscode",
 		"usernamehw.errorlens",
 		"visualstudioexptteam.vscodeintellicode",
-		"yzhang.markdown-all-in-one",
-		"github.copilot",
-		"karyan40024.gotmpl-syntax-highlighter"
+		"wenfangdu.jump",
+		"yzhang.markdown-all-in-one"
```

Comment:
I sorted some, added...

* GitHub.copilot-chat
* wenfangdu.jump

...and removed...

* coenraads.bracket-pair-colorizer-2: this plugin is deprecated as the functionality is now baked-in vscode. 
* tabnine.tabnine-vscode (copilot is just better)